src/main/java/gov/cabinetoffice/gap/adminbackend/services/GrantMandatoryQuestionService.java

```diff
@@ -33,14 +34,52 @@ public class GrantMandatoryQuestionService {
 
     private final SchemeService schemeService;
 
+    private final ZipService zipService;
+
     public List<GrantMandatoryQuestions> getGrantMandatoryQuestionBySchemeAndCompletedStatus(Integer schemeId) {
         return ofNullable(grantMandatoryQuestionRepository.findBySchemeEntity_IdAndCompletedStatus(schemeId))
                 .orElseThrow(() -> new NotFoundException(
                         String.format("No completed mandatory questions with ID %s was found", schemeId)));
```

Comment:
Not convinced `ofNullable` actually does anything here, this method would just return an empty list
```suggestion
        return grantMandatoryQuestionRepository.findBySchemeEntity_IdAndCompletedStatus(schemeId);
```